governance.md

```diff
@@ -70,6 +65,16 @@ The GB may invite guests to participate in consideration of specific topics (but
 may not participate in any voting matters).
 
 Primary activities of the GB shall be, but not limited to:
+* represent the broad FOSS community of India
+* ensure long term survival and growth of the FOSS United community
+* set long term goals for the FOSS United community
+* vote and approve the budget for the year
```

Comment:
Does this also mean that the GB is legally liable? You seem to be giving "real" powers to the Governing Board without also making it clear whether or not they will be legally liable for the Foundation's activities.

One of the primary reasons why we didn't go this far with the MVP charter is that "NGOs" that follow the structure you propose commonly have "fixed" Governing Boards that haven't changed in many years. I personally can't name a single Indian NGO that

- elects a Governing Board regularly, e.g., each year
- places fiscal and legal responsibility on them

can you name any?